test/featureflags.go

```diff
@@ -13,6 +13,39 @@ import (
 	"knative.dev/pkg/system"
 )
 
+var spireFeatureGates = map[string]string{
+	"enable-nonfalsifiability": "spire",
+	"enable-api-fields":        "alpha",
+}
```

Comment:
spire requires both of these feature flags to be enabled?

Reply:
what if I set `"enable-nonfalsifiability": "spire"`, but `"enable-api-fields": "stable"`?

Reply:
If enable-api-fields is not alpha, we get an error and the "enable-nonfalsifiability" will be set to the default i.e. no enforcement.
https://github.com/tektoncd/pipeline/blob/2a15755e9331156631aef8e382122b4bf685a131/pkg/apis/config/feature_flags.go#L221